environs/manual/sshprovisioner/sshprovisioner.go

```diff
@@ -185,7 +185,13 @@ func checkProvisioned(host string) (bool, error) {
 // detect the OS series and hardware characteristics.
 const detectionScript = `#!/bin/bash
 set -e
-lsb_release -cs
+os_id=$(grep '^ID=' /etc/os-release | tr -d '"' | cut -d= -f2)
```

Comment:
given we have set -e, does this fail if /etc/os-release doesn't exist, causing grep to return nonzero? (Or is the pipeline enough to prevent that)